Packages/Primitives/Sources/Extensions/Color+Primitives.swift

```diff
@@ -0,0 +1,14 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+import Foundation
+import SwiftUI
+
+public extension Color {
+    static func random(randomOpacity: Bool = false) -> Color {
```

Comment:
```suggestion
    static func random(randomOpacity: Bool = false) -> Color {
```

split into two method: `random()` and `randomOpacity()` so you call `color.random().randomOpacity()`